external/storm-kafka-migration/src/main/java/org/apache/storm/kafka/migration/KafkaTridentSpoutMigration.java

```diff
@@ -69,24 +70,40 @@ public String toString() {
         }
     }
 
+    /**
+     * Get value for key. Error if value is null or not the expected type.
+     */
+    public static <T> T getOrError(Map<String, Object> conf, String key) {
```

Comment:
Not a big deal (because the storm-kafka-migration module is just a utility), but getOrError() looks like better to place to separate class to decouple two independent classes.  Just a 2 cents and I'm OK even we keep it as it is.